include/oneapi/mkl/dft/detail/commit_impl.hpp

```diff
@@ -67,6 +77,20 @@ class commit_impl {
 
     virtual void commit(const dft_values<prec, dom> &) = 0;
 
+    inline std::int64_t get_workspace_external_bytes() {
+        return external_workspace_helper_.get_rqd_workspace_bytes(*this);
+    };
+
+    // Set workspace by be overrided for any backend that actually uses the workspace.
```

Comment:
Not sure what you meant here, there is a typo I think?

Reply:
I've reworded this slightly in [1da13ee](https://github.com/oneapi-src/oneMKL/pull/411/commits/1da13eef0fbb663fcadc1abbcc15ad930f977589).